assets/js/romo/date.js

```diff
@@ -78,6 +78,15 @@ var RomoDate = {
     return RomoDate.daysInMonth(d) - d.getDate() + 1;
   },
 
+  daysDiff: function(firstDate, secondDate) {
+    var diff = (firstDate.getTime() - secondDate.getTime()) / 864e5; // 1000 * 60 * 60 * 24
```

Comment:
@jcredding did not know you could represent integers that way - cool!

Reply:
@jcredding I'd prefer casting the given "dates" to actual dates like the method above does.  This keeps the api consistent and I think allows you to pass strings directly.